<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="leftBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>CuttingPlan</string>
       </property>
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_NewCuttingPlan">
            <property name="text">
             <string>New Plan</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_OpenCuttingPlan">
            <property name="text">
             <string>Open Plan</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_AddCuttingPlanRequest">
            <property name="text">
             <string>Add Row</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinKerf">
            <property name="suffix">
             <string notr="true" comment="mm"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="singleStep">
             <number>0</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ClearCuttingPlan">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="inputFileLabel">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableInput">
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Hossz</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mennyiség</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Műveletek</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="midBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="midBoxPage1">
        <attribute name="title">
         <string>Optimization</string>
        </attribute>
        <layout class="QVBoxLayout" name="midLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_Optimize">
             <property name="text">
              <string>Optimize</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_Finalize">
             <property name="text">
              <string>Finalize</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableResults">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Rod#</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Category</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Length</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Kerf</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Waste</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>StorageAudit</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="btn_StorageAudit">
             <property name="text">
              <string>Audit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblAuditStatus">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_Relocate">
             <property name="text">
              <string>ReLocate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableStorageAudit">
           <column>
            <property name="text">
             <string>Anyag</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Vonalkód</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tárhely</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Elvárt (Picking)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tényleges (Audit)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Hiányzó</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Státusz</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>RelocationOrder</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="relocateQuickList"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Stock</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="btn_AddStockEntry">
             <property name="toolTip">
              <string notr="true" extracomment="Egy meglévő anyag első bevitele a készletbe. Akkor használd, ha az anyag eddig nem szerepelt a raktárban."/>
             </property>
             <property name="text">
              <string>Új Tétel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableStock">
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Barcode</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Length (mm)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Shape</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Quantity</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Storage</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Comment</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Actions</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Leftovers</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btn_AddLeftoverStockEntry">
             <property name="text">
              <string>Új Tétel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_LeftoverDisposal">
             <property name="text">
              <string>Selejtezés</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableLeftovers">
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Barcode</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Reusable ID	</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Length (mm)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Shape</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Source</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Reusable</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Storage</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Actions</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
